examples/AN00239/README.rst

```diff
@@ -15,7 +11,8 @@ control of which debug messages are enabled.
 Required tools and libraries
```

Comment:
Does the python script check this section?  If we're not going to list tool dependencies here, then we should rename the section to 'Depends upon' or something similar.

Reply:
The Python script doesn't check any of the README files apart from the top-level one. I wanted to add a check for all the README file, it will be just a basic check with rstcheck to ensure the file is correctly formatted.

How do you want to rename the section exactly? "Software dependencies"?

Reply:
'Software dependencies' sounds good to me.